_pages/policies/travel-and-leave-policies/travel-guide-faq.md

```diff
@@ -295,6 +295,8 @@ If you'd like to schedule personal travel that's incidental to your work trip (i
 
 All official international travel taken by GSA employees, regardless of funding source, also requires Salesforce event approval. Please reach out to michael.proul@gsa.gov at least 7 weeks in advance in order to coordinate this.
 
+Teleworking from locations outside the United States while on personal travel--i.e., the "working vacation"--is not allowed.
```

Comment:
I think we can simplify this to "Teleworking from locations outside the U.S. is not allowed." (Ideally with a link to the relevant telework guidance)